### ISSUE TITLE
Admin content list filters and paging for contained list content items

### ISSUE THREAD

**Issue by scleaver** [OrchardCMS/OrchardCore#10826]
### Is your feature request related to a problem? Please describe.
In admin, contained lists (think posts on blog) don't have any filters, numbering (count) or more advanced paging (number of pages etc). If you have significant number of contained list items or want to implement so filtering options this becomes a problem.

### Describe the solution you'd like
The same options that non-contained content lists have.

### Describe alternatives you've considered
Currently you have to list the content type outside of the contained list in order to get the filtering options if you need them.

**Comment by Skrypt** [comment]
I think it is missing the UI and it is not consistent with the rest. Never took time to do that part because it is a little more tricky than what it looks like.

**Comment by deanmarcussen** [comment]
Yes @Piedone added display text filtering some time ago, but as @Skrypt says it is trickier than it looks.

Currently the related query service is used for both the front end query when displaying, and the admin query.

My suggestion would be, that to achieve other filtering, to seperate the front end and admin queries, rather than trying to force fit logic around filtering options.

See also https://github.com/OrchardCMS/OrchardCore/issues/8367 

> The tricky part is that you should be able to reorder the content items in the list when it is allowed too.

I would suggest to disable ordering when a filter is applied (I think it already does this)
